### ISSUE TITLE
Manual RangeInclusive implementation needs a space when using floats.

### ISSUE THREAD

**Issue by CGMossa** [rust-lang/rust-clippy#6315]
Clippy gave me this message:

```rust
warning: manual `!RangeInclusive::contains` implementation
   --> src\misc.rs:223:40
    |
223 |     if (log_p && p > 0.) || (!log_p && ((p < 0.) || (p > 1.))) {
    |                                        ^^^^^^^^^^^^^^^^^^^^^^ help: use: `!(0...=1.).contains(&p)`
    |
    = note: `#[warn(clippy::manual_range_contains)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#manual_range_contains
```

But this `!(0...=1.).contains(&p)` is not valid,
it needs a space i.e. `!(0. ..=1.).contains(&p)`.

I don't know how to report this in a better way, but here you go.

### Meta

- `cargo clippy -V`: e.g. clippy 0.0.212 (f455e46 2020-06-20)
- `rustc -Vv`:
```
Microsoft Windows [Version 10.0.19041.572]
(c) 2020 Microsoft Corporation. All rights reserved.

[redacted]>cargo clippy -V
clippy 0.0.212 (1773f60e 2020-11-08)

[redacted]>rustc -Vv
rustc 1.49.0-nightly (1773f60ea 2020-11-08)
binary: rustc
commit-hash: 1773f60ea5d42e86b8fdf78d2fc5221ead222bc1
commit-date: 2020-11-08
host: x86_64-pc-windows-msvc
release: 1.49.0-nightly
```
I'm not including a backtrace, as there was no panic to trace.

Thanks!

**Comment by giraffate** [comment]
I will prepare a fix.
